src/analyses/ai.cpp

```diff
@@ -480,6 +484,7 @@ bool ai_recursive_interproceduralt::visit_edge_function_call(
       p_call,
       callee_function_id,
       l_begin,
+      nullptr,
```

Comment:
Why is this one not given?

Reply:
Because it is only used on function call return edges, this is a call, so `p_call` already has the caller info.